addons/mail/models/discuss/discuss_channel.py

```diff
@@ -454,6 +461,14 @@ def write(self, vals):
                         channels=failing_channels.mapped("name"),
                     )
                 )
+        if "active" in vals:
+            if failing_channels := self.filtered(lambda channel: channel.channel_type in ["channel", "group"] and not channel.self_member_id.channel_role == "owner" and not self.env.user.has_group("base.group_system")):
+                raise UserError(
+                    self.env._(
+                        "Cannot deactivate channels: %(channels)s as you are not the owner.",
```

Comment:
The condition as it is written will also prevent from activating them, which is not clear from the error message.